server/core/region.go

```diff
@@ -214,6 +215,14 @@ func (r *RegionInfo) Clone(opts ...RegionCreateOption) *RegionInfo {
 	return region
 }
 
+// GetBuckets returns the buckets of the region.
+func (r *RegionInfo) GetBuckets() *metapb.Buckets {
+	if r == nil {
```

Comment:
When will it happen?

Reply:
this is the first step,  sync and some api will use it, I will split them into different pr.

Reply:
I mean `r == nil`